doc(readme): update github and npm version badges

GitToTheHub opened the pull request:
I find it better, when you can see the current `package.json` version of the repository in the title. I used `shields.io` for it. To not be confused with the current released npm package, I created a section `Stable Release` which shows the npm version. I removed the nodei.co badge because it was hard to read and showed `npm install cordova-android` though you should add a platform with `cordova platform add android`.

<!--
Please make sure the checklist boxes are all checked before submitting the PR. The checklist is intended as a quick reference, for complete details please see our Contributor Guidelines:

http://cordova.apache.org/contribute/contribute_guidelines.html

Thanks!
-->

### Platforms affected



### Motivation and Context
<!-- Why is this change required? What problem does it solve? -->
<!-- If it fixes an open issue, please link to the issue here. -->



### Description
<!-- Describe your changes in detail -->



### Testing
<!-- Please describe in detail how you tested your changes. -->



### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

GitToTheHub committed:
Enhance README with version badges

- Added current package version as badge in title
- Removed nodei.co badge
- Added npm version badge via shields.io in section `Stable Release`

erisu commented on README.md:
```diff
@@ -19,9 +19,7 @@
-# Cordova Android
-
-[![NPM](https://nodei.co/npm/cordova-android.png)](https://nodei.co/npm/cordova-android/)
+# Cordova Android ![GitHub package.json version](https://img.shields.io/github/package-json/v/apache/cordova-android)
```
```suggestion
# Cordova Android

[![NPM](https://nodei.co/npm/cordova-android.svg?style=flat&data=n,v,u&color=blue)](https://www.npmjs.com/package/cordova-android)
```

erisu replied:
Here's another possible example: https://gist.github.com/erisu/2536f97976e64165c00a781888b0b6b4

Just ignore the two that displays as text. Some reason its not rendering correctly in the gist but renders OK in the repo.

Below the title, it shows the latest release on npm. The next row shows development related badges.

GitToTheHub committed:
Update README.md badges and remove Stable Release section

. Updated badges in README.md for npm version and GitHub version by using `shields.io` for the badges, because it has a more open license (uses `CC0-1.0`) than `nodei.co` and offers labels, which `nodei.co` does not support
- Removed the Stable Release section again, the badge is now shown directly underneath the main title
